guardian/riff-raff
# Adopt log shipping via DevX logs

Requires https://github.com/guardian/deploy-tools-platform/pull/738.

## What does this change?
Adopts https://github.com/guardian/devx-logs to ship application logs to Central ELK. This change allows us to remove an application dependency (https://github.com/guardian/kinesis-logback-appender), ultimately reducing the volume of code in the repository.

> [!IMPORTANT]
> We'll lose the `buildNumber` log marker with this change.

## How to test
- [Deploy to CODE](https://riffraff.code.dev-gutools.co.uk/deployment/view/134da91a-0eac-4805-b6e7-6458e82e4b1d)
- Observe [logs appearing in Central ELK](https://logs.gutools.co.uk/s/devx/goto/d476a6b0-9ac3-11ee-8f88-e3f4abf89c65)

## Rollout
- [x] Add required tags to the ASG (https://github.com/guardian/deploy-tools-platform/pull/738)
- [x] Cycle the instance; the new one will have the new tag
   ```bash
   ssm cmd -t riff-raff,PROD -p deployTools --cmd "curl --silent -X POST localhost:9000/requestInstanceRotation"
   ```
